sklearn/linear_model/_sgd_fast.pyx

```diff
@@ -21,7 +21,7 @@ from numpy.math cimport INFINITY
 cdef extern from "_sgd_fast_helpers.h":
     bint skl_isfinite(double) nogil
 
-from ..utils._weight_vector cimport WeightVector
+from ..utils._weight_vector cimport WeightVector64 as WeightVector
 from ..utils._seq_dataset cimport SequentialDataset64 as SequentialDataset
```

Comment:
To follow-up with @jeremiedbb's comment (https://github.com/scikit-learn/scikit-learn/pull/13358/commits/50de2663befcbd755db0fa62535dfdb71ed27c4a#r261996527): to me, this looks fine as the import is done similarly for `SequentialDataset`; am I missing something?

Reply:
Up @jeremiedbb ?

Reply:
I cannot see the original comment but this looks fine to me as well.